src/script/components/InputLevel.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import TestPage from 'Util/test/TestPage';
+import InputLevel, {InputLevelProps, MAX_AUDIO_BULLETS} from 'Components/InputLevel';
+
+class InputLevelTestPage extends TestPage<InputLevelProps> {
+  constructor(props?: InputLevelProps) {
+    super(InputLevel, props);
+  }
+
+  getActiveInputLevelBullets = () => this.get('li.input-level-bullet-active');
```

Comment:
Let's decouple that from CSS and use data-uie-names

Reply:
I don't like spamming the code with `data-uie-names` but I will do it this time and come up with a better solution next time.

Reply:
https://www.npmjs.com/package/babel-plugin-remove-jsx-attributes

Reply:
And then in `babel.config.js`
![image](https://user-images.githubusercontent.com/1138344/106929602-da099500-6714-11eb-955e-d6bb88575cd8.png)

Reply:
I think there's nothing wrong in using `data-uie-name`s and `value`s and I don't see it as clutter. It makes it easy to select components, it makes it easy to see what is actually selected in a testcase, it doesn't break when you change the CSS and QA can use it as well.

Plus, how would you select a component that doesn't have a class or id attached?

Reply:
1. I don't think that a selection with `li[data-uie-name="input-level-bullet-active"]` is easier than `li.input-level-bullet-active`. Both are easy to write.
1. It's more cluttered because `<li class="input-level-bullet-active">` now becomes `<li class="input-level-bullet-active" data-uie-name="input-level-bullet-active">`.
1. Our QA team doesn't need to select this component because we now have a test for it.
1. Instead, creating a `data-uie-name` can now implicitly mean that our QA team is using that component (even if they don't) which makes us afraid of changing its name or removing it.
1. Ofcourse the `data-uie-name` doesn't break when changing the CSS name but it breaks when changing the `data-uie-name` (which may be needed if the component gets renamed, i.e. from "input-level" to "audio-input-level").
1. Adding `data-uie-name` attributes to select components was an effort made to simplify the access for our QA team. Since our QA team doesn't need to access this component from an E2E test, we can use tools like [resq](https://github.com/baruchvlz/resq) to select a React component.

Reply:
But QA requested data-uie's for the exact same reason we are using them now. Decoupling from styling and classes and ids.

1. That's true for simple components but it makes a difference for complex ones.
2. Okay, but worth it IMHO
3. Having a unit test doesn't mean QA doesn't want to write an E2E test on top.
4. Good point, should we name our data attributes differently? (potentially even more attributes then)
5. Of course but changing CSS is more likely and that's why QA requested data-uies.
6. resq doesn't help because you I believe you can't transfer this object to enzyme.

I'm all up for a better solution or improvements but until then, data attributes seem to be state of the art.

Reply:
Ah damn, food got cold 😭